variables.tf

```diff
@@ -58,4 +58,10 @@ variable "fr_product" {
   default = "fees-register"
 }
 
-variable "aks_subscription_id" {}
\ No newline at end of file
+variable "sku" {
+  type        = string
+  default     = "Premium"
```

Comment:
Do you want Premium SKU in all envs? For cost saving, I would only have Premium in Prod/AAT